src/controllers/coach/index.js

```diff
@@ -189,6 +190,33 @@ const searchUserAsCoach = async (req, res) => {
   }
 }
 
+const addWorkout = async (req, res) => {
+  try {
+    const {
+      user,
+      body: { title, content, lang },
+    } = req
+
+    if (!acceptedLanguagesValue.includes(lang))
+      throw new Error('Lang is invalid')
+
+    const language = await getLangByISO(lang)
+    const workout = await createWorkout(
+      user._id,
+      language._id.toString(),
+      title,
+      content,
+    )
+
+    res.status(201).json(workout)
```

Comment:
Why 201 instead of  200 ?

Reply:
https://restfulapi.net/http-status-codes/